src/App.tsx

```diff
@@ -1,25 +1,120 @@
 import './App.scss';
-
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
 
 export const App = () => {
+  const [newToDo, setNewToDo] = useState({
+    title: '',
+    user: '',
+  });
+  const todosCopy = [...todosFromServer];
+  const copy = [...usersFromServer];
+
+  const [toDoList, setToDoList] = useState(todosCopy);
+
+  function submitHandler(event: React.FormEvent<HTMLFormElement>) {
+    event.preventDefault();
+    const maxId = Math.max(...todosCopy.map(todo => +todo.id));
+    const eventUser = (event.currentTarget[1] as HTMLInputElement).value;
+    const eventTitle = (event.currentTarget[0] as HTMLInputElement).value;
+
+    if (eventUser === '0') {
+      setNewToDo(prevState => ({
+        title: prevState.title,
+        user: 'alarm',
+      }));
+      setTimeout(() => setNewToDo(prevState => ({
+        title: prevState.title,
+        user: '',
+      })), 1000);
+    } else if (eventTitle === '') {
```

Comment:
don't forget about empty space in an input value, so please use .trim()

Reply:
spaces should be allowed )
<img width="1280" alt="Screenshot 2023-06-11 at 21 47 05" src="https://github.com/mate-academy/react_add-todo-form/assets/20277989/8bda2dd8-4f85-42f7-9600-a3c2e3e62446">